x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/index.ts

```diff
@@ -196,6 +208,50 @@ export class APMEventClient {
     });
   }
 
+  async logEventSearch<TParams extends APMLogEventESSearchRequest>(
+    operationName: string,
+    params: TParams
+  ): Promise<TypedLogEventSearchResponse<TParams>> {
+    // Reusing indices configured for errors since both events and errors are stored as logs.
+    const index = processorEventsToIndex([ProcessorEvent.error], this.indices);
+
+    const filters: ESFilter[] = [
+      {
+        terms: {
+          ['event.kind']: ['event'],
```

Comment:
same here: `event.kind` should be a constant

Reply:
Since `event.kind` was removed, I think we can resolve this conversation.